src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  margin-top: 0;
+}
```

Comment:
You should remove all margins in `body` tag

Reply:
that was the point! thank you!